src/styles/abstracts/_mixins.scss

```diff
@@ -44,3 +44,19 @@
   justify-content: center;
   align-items: center;
 }
+
+@mixin button-bg($bg) {
+  background: $bg;
+
+  padding: rem(5px 10px);
+  border-radius: rem(3px);
+  text-decoration: none;
+  
+  &:hover {
+    background: darken($bg, 8%);
```

Comment:
Wouldn't it be better to name this colour if it was reused in many places?

Reply:
the `$bg` color depends on the button's background color we want. So, I would not specify a single color for it.